test/features/email/presentation/controller/single_email_controller_test.dart

```diff
@@ -252,4 +261,39 @@ void main() {
       },
     );
   });
+
+  group('calendar event reply -', () {
```

Comment:
```suggestion
  group('calendar event reply test', () {
```

Reply:
Same as [above](https://github.com/linagora/tmail-flutter/pull/2815#discussion_r1574208778). Consider the output of the tests, and if that's what you want, I'll make the change.

Reply:
Done in:
- https://github.com/linagora/tmail-flutter/pull/2803